docs/2_2_common_mechanisms.adoc

```diff
@@ -627,14 +627,14 @@ The importer calls <<fmi3GetClock>> to inquire the clock activation state.
 |<<fmi3SetClock>> in <<EventMode>>,
 <<fmi3ActivateModelPartition>> in <<ClockActivationMode>>,
 <<fmi3GetInterval>> in <<InitializationMode>>,
-<<fmi3GetInterval>> in both <<EventMode>> and <<ClockActivationMode>> if and only if this Clock ticked
+<<fmi3GetInterval>> in both <<EventMode>> and <<ClockActivationMode>> if and only if this Clock ticked using <<clocksTicked,`clocksTicked == fmi3True`>>
 |simulation of the behavior of a control algorithm with variable execution time, generation of pulse sequences
```

Comment:
The word "using" is not clear enough here Andreas... Maybe "when XXXX is satisfied in the callback function."